src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -425,6 +487,7 @@ def _set_names(self, values: list) -> None:
         WarningMessage.index_to_pandas_warning("_set_names")
         self.to_pandas()._set_names(values)
 
+    # TODO: SNOW-1458122 implement names
```

Comment:
There's nothing to really do here right once `_set_names` and `_get_names` is implemented?

Reply:
I think so? I'm not reallu sure what the scope of the task is, I just put JIRA tickets in to make it easier to reference.

Reply:
this line is not needed.